.env

```diff
@@ -0,0 +1,7 @@
+DB_CONNECTION_STRING=postgresql://postgres:123@localhost:5432/afcfitness
+APP_SECRET_KEY=wegwegiwewefwefe
+GMAPS_API_KEY=AIzaSyAKDHnR80au2cURkbiCZyKg061A1cZt3MY
+OPENAI_API_KEY=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
```

Comment:
You should never commit your API keys on a public repo (even in a private repo) because it will stay in your GIT history.